Stratis.Bitcoin/Utilities/Stopwatch.cs

```diff
@@ -0,0 +1,83 @@
+﻿using System;
+
+namespace Stratis.Bitcoin.Utilities
+{
+    /// <summary>
+    /// Replacement for <see cref="System.Diagnostics.Stopwatch"/> class that allows the caller 
```

Comment:
Replacement for **the**  Stopwatch class that allows the caller to use **a** convenient way of calling the watch with **the**  <c>using</c> **statement**...